pkg/og/structs/flamebearer/flamebearer.go

```diff
@@ -82,7 +82,7 @@ type FlamebearerMetadataV1 struct {
 	Format string `json:"format"`
 	// Name of the spy / profiler used to generate the profile, if any.
 	SpyName string `json:"spyName"`
-	// Sample rate at which the profiler was operating.
+	// Sample rate at which the profiler was operating. Note: This unit is not related to the Units field and is measured in samples per second.
 	SampleRate uint32 `json:"sampleRate"`
```

Comment:
I am not too sure if this is the right type and definition. I think it would be better if we would signal the sampleRate per SampleUnit. But for this we would need to change the field to be a float.

The default `100 Hz`, would become `0.00001 per nanosecond`. Any insights that you have on this @korniltsev